frontend/public/components/service-instance.tsx

```diff
@@ -43,9 +44,8 @@ const ServiceInstanceDetails: React.SFC<ServiceInstanceDetailsProps> = ({obj: si
         </div>
         <div className="col-sm-6">
           <dl className="co-m-pane__details">
-            <dt>External Name</dt>
-            {/* TODO:  add link */}
-            <dd>{si.spec.clusterServiceClassExternalName || si.spec.serviceClassExternalName}</dd>
+            <dt>Service Class</dt>
+            <dd><ResourceLink kind="ClusterServiceClass" displayName={classDisplayName} title={classDisplayName} name={si.spec.clusterServiceClassRef.name} /></dd>
```

Comment:
We need to check for the existence of that value... This could be a runtime error (white screen) otherwise.

```jsx
const clusterServiceClassName = _.get(si, 'spec.clusterServiceClassRef.name');
// ...
<dd>
  {clusterServiceClassName
    ? <ResourceLink kind="ClusterServiceClass" displayName={classDisplayName} name={clusterServiceClassName} />
    : classDisplayName}
</dd>
```

I would leave off the `title` attribute. It's not helpful here.